spec/feature/authenticated_user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+describe "an authenticated user" do
+
+  it "can view their own page" do
+    user = User.create(first_name: "Alice",
+                       last_name: "Smith",
+                       email: "rich.shea@gmail.com",
+                       password: "password")
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
```

Comment:
I would be ok with ignoring this hound error, don't see how to fix without hound taking issue...